### PR TITLE
Add test 'Tax rules bulk actions'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Add test 'Tax rules bulk actions'
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | TEST_PATH="functional/BO/11_international/03_taxes/taxRules/03_bulkActions.js" URL_FO=yourShopURL npm run specific-test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/21517)

### CODE REVIEW

File: tests/UI/campaigns/functional/BO/11_international/03_taxes/taxRules/03_bulkActions.js [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        const textResult = await taxRulesPage.getTextColumnFromTable(page, 1, 'name');
+        await expect(textResult).to.contains(test.args.taxRule);
+      });
+
+      it(`should ${test.args.action} tax rules with bulk actions and check Result`, async function () {
+        await testContext.addContextItem(this, 'testIdentifier', `bulk${test.args.action}`, baseContext);
+
+        const textResult = await taxRulesPage.changeEnabledColumnBulkActions(page, test.args.enabledValue);
+
+        /* Successful message is not visible, skipping it */
```
Comment (SimonGrn) [inline tests/UI/campaigns/functional/BO/11_international/03_taxes/taxRules/03_bulkActions.js:137]: Is this a known issue ? We need to keep track of this to be sure to re-enable it when it's fixed

File: tests/UI/pages/BO/international/taxes/taxRules/index.js [hunk 2]
```diff
@@ -313,6 +315,25 @@ class TaxRules extends BOBasePage {
...
+    // Select all rows
+    await this.bulkSelectRows(page);
+
+    // Click on Button Bulk actions
+    await page.click(this.bulkActionMenuButton);
+
+    // Click to change status
+    await this.clickAndWaitForNavigation(page, enable ? this.bulkEnableLink : this.bulkDisableLink);
+    /* Successful message is not visible, skipping it */
+    //return this.getTextContent(page, this.alertSuccessBlock);
```
Comment (SimonGrn) [inline tests/UI/pages/BO/international/taxes/taxRules/index.js]: Same here, we need to keep track of this.

**Review by SimonGrn — DISMISSED** [review]

File: tests/UI/pages/BO/international/taxes/taxRules/index.js (hunk 2 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/international/taxes/taxRules/index.js]: Okey :+1:

File: tests/UI/campaigns/functional/BO/11_international/03_taxes/taxRules/03_bulkActions.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/11_international/03_taxes/taxRules/03_bulkActions.js:137]: Okey, I will check if there is an issue :+1:

**Review by NeOMakinG — DISMISSED** [review]

**Review by SimonGrn — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by SimonGrn — APPROVED** [review]

**Pull request merged**